frontend/docs/seo-guide.md

```diff
@@ -0,0 +1,359 @@
+
+#### Page Speed
+
+Page speed is a measurement of how fast the content on your page load. You can evaluate your page speed with [Google's PageSpeed Insights.](https://developers.google.com/speed/pagespeed/insights/) Google has indicated page speed **is one of the signals used by its algorithm to rank pages**.
+
+You can find some of the many ways to increase your page speed: 
+
+- Use **Gzip**, a software application for file compression, to reduce the size of your CSS, HTML, and JavaScript files.
+- **Minify CSS, JavaScript, and HTML**.
+- Remove **render-blocking JavaScript**.
+- Leverage **browser caching**.
+
+_Why:_
+> Page speed is a measurement of how fast the content on your page loads. [Read more...](https://moz.com/learn/seo/page-speed)
+
+### 2.2 Content
+
+#### General
+
+Content is the most important thing in SEO. Keep in mind:
+
+- New content is important! **Keep your page active** and Google will reward it.
+- Content does **not have orthographic or serious grammatical errors**.
+- All your pages should have **at least 300 words**. If your web has less content, is it really necessary?
+- **Text/HTML ratio** is not a direct ranking factor for search engines but there are many factors related to the ratio that indicate best SEO practices and thus may indirectly result in higher search rankings. A good text to HTML ratio is about from 25 to 70 percent.
+
+#### Title
+
+This element especifies **the title of a webpage**. Title tags are displayed on search engine results pages (SERPs) as the clickable headline for a given result, and are important for usability, SEO, and social sharing.
+
+- Is about **50-60 characters long**, including spaces.
+
+- Is concise and **contain your best keywords**.
+
+- Each page should have its own **exclusive title**.
+
+- Here you can find an (snippet optimizer)[http://www.seomofo.com/snippet-optimizer.html] that allows you to see how your titles look on Google and other search results.
+
+_Why:_
+> Title tags are a major factor in helping search engines understand what your page is about, and they are the first impression many people have of your page. [Read more...](https://moz.com/learn/seo/title-tag)
+
+#### Images
+
+Almost every content has to have any image in it. But, how this affect SEO? Well, without the proper optimization, however, you’re wasting a valuable SEO asset. Plus, the larger the file size, the longer it takes your page to load, and the higher your bounce rate will be.
+
+- Be sure that your images are no larger than they need to be, that they are in the right file format.
+  _Why:_
+  > Your pages load time will vary based on the number and size of your images.
+
+- All images must have an `alt` and `title` attributes implemented that describes its content.
+  _Why:_
+  > ALT text attaches a description to your pictures so that they show up in Google and other search engine's image results. [Read more...](https://yoast.com/image-seo-alt-tag-and-title-tag-optimization/)
+
```

Comment:
Another important thing is to specify when to use an SVG or an PNG. When the image is an icon not a picture of something. SVG is the best choice

Reply:
:warning: image is a bad example for alts (also not recommended for SEO, as it does not represent the content

Reply:
@emanuelcasco